storage/mysql/storage_test.go

```diff
@@ -1271,9 +1271,9 @@ func ensureAllLeafHashesDistinct(leaves []trillian.LogLeaf, t *testing.T) {
 	// or pretty much any kind of usable data structures we could do this properly.
```

Comment:
Maybe update the comment to indicate which sort of 'hashes' are involved?  (It's probably a good idea to always disambiguate the word 'hash' when it's used in the context of a leaf...)